.eslintrc

```diff
@@ -16,6 +16,7 @@
     "curly": ["error", "multi-line"],
     "import/no-unresolved": [2, { "commonjs": true }],
     "no-shadow": ["error", { "allow": ["req", "res", "err"] }],
+    "linebreak-style": "off",
```

Comment:
I don't think you are meant to do this?

Reply:
I does not harm any code.
The eslint file is there to turn of any thing we don't want eslint to look at. And CRLF is crazy on my system. Even when I change to LF, when I reopen the file it goes back to CRLF. Since I added that no red lines.
Remember it does not make the code less or error prone.

Reply:
@okobaba1 I think you are right @UggoPrince Please, run this on your terminal to prevent git from reconverting all `LF` to `CRLF`: ``` git config --global core.auto crlf false ```